test/create_db.sh

```diff
@@ -22,33 +53,47 @@ mysql $dbconn -e "SET GLOBAL max_connections = 500;"
 for dbenv in $DBENVS; do
   db="boulder_sa_${dbenv}"
 
+  print_heading "Checking if "$db" exists"
   if mysql $dbconn -e 'show databases;' | grep $db > /dev/null; then
-    echo "Database $db already exists - skipping create"
+    echo ""$db" already exists - skipping create"
   else
-    create_script="drop database if exists \`${db}\`; create database if not exists \`${db}\`;"
-
-    mysql $dbconn -e "$create_script" || die "unable to create ${db}"
+    echo ""$db" doesn't exist - creating"
+    create_empty_db
+  fi
 
-    echo "created empty ${db} database"
+  # determine which db_path to use
+  if [[ "$BOULDER_CONFIG_DIR" = "test/config-next" ]]
+  then
+    dbpath="./sa/_db-next/"
+    migrations="./sa/_db-next/migrations"
+  else
+    dbpath="./sa/_db/"
+    migrationspath="./sa/_db/migrations"
   fi
 
-  goose -path=./sa/_db/ -env=$dbenv up || die "unable to migrate ${db} with ./sa/_db/"
-  echo "migrated ${db} database with ./sa/_db/"
-
-  if [[ "$BOULDER_CONFIG_DIR" = "test/config-next" ]]; then
-    nextDir="./sa/_db-next/"
-
-    # Goose exits non-zero if there are no migrations to apply with the error
-    # message:
-    #   "2016/09/26 15:43:38 no valid version found"
-    # so we only want to run goose with the nextDir if there is a migrations
-    # directory present with at least one migration
-    if [ $(find "$nextDir/migrations" -maxdepth 0 -type d -not -empty 2>/dev/null) ]; then
-      goose -path=${nextDir} -env=$dbenv up || die "unable to migrate ${db} with ${nextDir}"
-      echo "migrated ${db} database with ${nextDir}"
-    else
-      echo "no ${nextDir} migrations to apply"
-    fi
+  # populate list of migration files for $dbpath
+  get_migrations
+
+  # goose up to the latest schema present
+  apply_migrations
+
+  # latest schema is the last in the array of 
+  latest_schema="$(basename -- "${migrations[-1]}")"
```

Comment:
I think this should be `latest_migration`. Also the sentence ends without finishing. In general if a comment is a full sentence, I'd like it to start with a capital letter and end with a period.